go/client/fuse_status_osx.go

```diff
@@ -61,3 +70,28 @@ func KeybaseFuseStatus(bundleVersion string) keybase1.FuseStatus {
 
 	return st
 }
+
+func mountInfo(fstype string) ([]keybase1.FuseMountInfo, error) {
+	out, err := exec.Command("/sbin/mount", "-t", fstype).Output()
```

Comment:
Ok. I was just worried parsing the output of a human readable command is
fragile and can break for reasons we can't anticipate.  I saw this when
shelling out to gpg.


On Thursday, November 19, 2015, Gabriel Handford <notifications@github.com>
wrote:

> In go/client/fuse_status_osx.go
> <https://github.com/keybase/client/pull/1362#discussion_r45384240>:
>
> > @@ -61,3 +70,28 @@ func KeybaseFuseStatus(bundleVersion string) keybase1.FuseStatus {
> >
> >  	return st
> >  }
> > +
> > +func mountInfo(fstype string) ([]keybase1.FuseMountInfo, error) {
> > +	out, err := exec.Command("/sbin/mount", "-t", fstype).Output()
>
> Looks like the safest thing to do is use the DiskArbitration library. I
> created a ticket for that here:
>
> https://keybase.atlassian.net/browse/CORE-2122
>
> I think in the meantime calling /sbin/mount is a good shortcut?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1362/files#r45384240>.
>